static/js/src/components/workshops/workshops.js

```diff
@@ -4,8 +4,10 @@ export default function(Workshops) {
   return {
     template: template,
     controllerAs: 'vm',
-    controller: function() {
-      this.workshops = Workshops.query();
+    controller: function($scope) {
```

Comment:
Trying to avoid passing in `$scope` whenever possible.  Angular 2.0 won't support it, and I'm trying to structure this code to be as 2.0 ready as possible.